cli/pack/common.go

```diff
@@ -346,6 +346,12 @@ func createEnv(opts *config.CliOpts, destPath string, cartridgeCompat bool) erro
 		cliOptsNew.App.WalDir = configure.VarWalPath
 	}
 
+	if cartridgeCompat {
+		cliOptsNew.App.InstancesEnabled = "."
+		cliOptsNew.App.IncludeDir = "."
```

Comment:
Not sure about this change. We do not pack tarantool include files and application is already build after packing. Why not to leave this default?

Reply:
Ok, fixed.